### PR TITLE
SC-3912 Produced percentile keywords from new constraint types.

### PR DESCRIPTION
**Pull request by jluhrs**
What used to be a simple read of a value now uses conversion tables and values read from TCS, making it more complicated.

### CODE REVIEW

File: modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala [hunk 1]
```diff
@@ -3,54 +3,153 @@
...
-        case WaterVapor.VeryDry => 1.0
-        case WaterVapor.Dry => 1.6
-        case WaterVapor.Median => 3.0
-        case WaterVapor.Wet => 5.0
-      case Site.GS => wv match
-        case WaterVapor.VeryDry => 2.3
-        case WaterVapor.Dry => 4.3
-        case WaterVapor.Median => 7.6
-        case WaterVapor.Wet => 10.0
+      case Site.GN =>
```
Comment (cquiroz) [inline modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala:29]: Do you think we should put things like this on db? Such that we can change them without rebuilding?

File: modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala [hunk 2]
```diff
@@ -3,54 +3,153 @@
...
+          NonEmptyList.of(0.35, 0.50, 0.75, 1.25),
+          NonEmptyList.of(0.35, 0.50, 0.70, 1.15),
+          NonEmptyList.of(0.31, 0.37, 0.45, 0.75)
+        )
+        .map(_.zip(NonEmptyList.of(20, 70, 85, 100)))
+    )
+
+  extension (iq: ImageQuality) {
+    def toPercentile(wv: Wavelength, elevation: Angle): Int = {
+      // Pickering approximation (thanks Andy Stephens)
```
Comment (cquiroz) [inline modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala:102]: What is pickering?
It seems this should go to `lucuma-core`

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala (hunk 2 above)
Comment (jluhrs) [reply modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala:102]: Pickering is a name. Ref: https://en.wikipedia.org/wiki/Air_mass_(astronomy)#Interpolative_formulas

File: modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala (hunk 1 above)
Comment (jluhrs) [reply modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala:29]: I don't know how frequently those values change. It is a question fro Andy.

**Pull request merged**